packages/shared/src/features/onboarding/hooks/useFunnelTracking.ts

```diff
@@ -151,20 +153,25 @@ export const useFunnelTracking = ({
 
   useEffect(
     () => {
-      trackFunnelEvent({ name: FunnelEventName.StartFunnel });
-      // todo: implement resume funnel event
+      const firstStepId = funnel?.chapters[0]?.steps[0]?.id;
+      const isResumedSession = session.currentStep !== firstStepId;
+
+      trackFunnelEvent({
+        name: isResumedSession
+          ? FunnelEventName.ResumeFunnel
+          : FunnelEventName.StartFunnel,
+      });
 
       return () => {
+        // todo: don't send this event if completed
```

Comment:
Is this something to be worked on from this PR?

Reply:
We need to do it before merge into main, but non strictly related to this task